internal/domain/container/container.go

```diff
@@ -81,6 +81,7 @@ type Container struct {
 	InternalHost                *string
 	ExternalHost                *string
 	State                       status.State
+	DeploymentStageId           string
```

Comment:
OPT: `DeploymentStageId` -> `DeploymentStageID`

CF https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Arf yes, I finished by writing `Id` instead of `ID`: is there some Go recommendations or is it subjective ?

Reply:
Yep, you can find a lot of info here -> https://github.com/golang/go/wiki/CodeReviewComments#initialisms